tests/avocado/test_nec-usb-xhci.py

```diff
@@ -52,4 +53,5 @@ def test_available_after_migration(self):
 
     def tearDown(self):
         self.vm.shutdown()
-        self.vm_dst.shutdown()
+        if self.vm_dst is not None:
+            self.vm_dst.shutdown()
```

Comment:
sure, this should be squashed before sending...

Reply:
btw this is why I'd prefer to have list of vms instead of `self.vm`, where `self.vms` would clean all registered vms... Anyway I know we want to start simple...